analysers/analyser_osmosis_indoor.py

```diff
@@ -102,11 +177,14 @@
         indoor_surfaces_other.indoor IN ('room', 'corridor', 'area') AND
         indoor_surfaces_other.geom && indoor_surfaces.geom AND
         indoor_surfaces_other.nodes && indoor_surfaces.nodes
+    LEFT JOIN indoor_pt_platforms ON
+        indoor_pt_platforms.geom && indoor_surfaces.geom
```

Comment:
`&&` is bbox intersection. You only do approximation with bbox intersection. Since there is few data and you already compute the polygons you can use a real intersection.

Reply:
indeed! actually, we don't even need polygons, what is really important is node overlap